docs/docs/tutorial/index.md

```diff
@@ -441,6 +466,16 @@ This opens a duplex connection and you should see messages indicating successful
 }
 ```
 
+:::tip
+You can use the `hydra-tui` to view the state of the node and perform actions
+as well. For example, to run the TUI for alice:
+
+```
+hydra-tui -k credentials/alice-node.sk
```

Comment:
Don't use the node keys in the hydra-tui. If we suggest to use it, we should point to the `alice-funds.sk`